README.md

```diff
@@ -101,18 +101,32 @@ Be aware that Cloud Shell has [several limitations][cloud-shell-limitations],
 in particular an inactivity timeout that will close running shells after 20
 minutes. Please consider it only for blueprints that are quickly deployed.
 
-## Blueprint Warnings and Errors
+## Blueprint Validation
```

Comment:
It would be my preference to greatly trim down this section in the base README.md and move most of the content to a dedicated doc somewhere under doc/. As it stands now we have 117 lines of documentation about validators on our landing page.

I still think it is important to highlight the feature in the base README.md and funnel people to the dedicated doc.